### PR TITLE
chore: addding unit test for missing validation

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes parts of #5320

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by maze-runnar** [comment]
@sachinchauhan2889 @MeghalBisht @ranjitkshah

File: tests/all/unit/api/validation/test_sessions_types.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import unittest
+from unittest import TestCase
+
+from app.api.schema.session_types import SessionTypeSchema
+
+
+class TestEventValidation(TestCase):
+    def test_validate_length_pass(self):
+        """
+        Events Validate Timezone - Tests if the function runs without an exception
```
Comment (MeghalBisht) [inline tests/all/unit/api/validation/test_sessions_types.py:10]: Session Type Validation

File: tests/all/unit/api/validation/test_sessions_types.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    def test_validate_length_pass(self):
+        """
+        Session Types Validate length - Tests if the function runs without an exception
+        :return:
+        """
+        schema = SessionTypeSchema()
+        data = {'length': '12:23'}
+        SessionTypeSchema.validate_length(schema, data)
+
+    def test_validate_length_format(self):
```
Comment (MeghalBisht) [inline tests/all/unit/api/validation/test_sessions_types.py:18]:
```suggestion
    def test_validate_length_exception(self):
```

File: tests/all/unit/api/validation/test_sessions_types.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        """
+        Session Types Validate length - Tests if the function runs without an exception
```
Comment (MeghalBisht) [inline tests/all/unit/api/validation/test_sessions_types.py:20]:
```suggestion
        Session Types Validate length - Tests if exception is raised
```

**Review by MeghalBisht — APPROVED** [review]
🥂

**Review by sachinchauhan2889 — APPROVED** [review]

**Review by ranjitkshah — APPROVED** [review]

**Pull request merged**